package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stripe/terminal-js",
-  "version": "1.0.0",
+  "version": "0.0.1",
```

Comment:
does this mean we're downgrading our terminal-js version? is that safe?

Reply:
JS doesn't have the same concept of versioning that Android/iOS does -- this version number will be displayed as the package version in NPM, but won't be the version number sent to Stripe.

Reply:
yeah, I'm just setting it down to something low so it's clear this is beta to anyone that stumbles into it. The version has no meaning until we publish it to npm and this won't be the version we send to readers.